### PR TITLE
Patching Liferay DXP

### PR DESCRIPTION
**Pull request by jhinkey**
Hey Rich,

Forwarding you these admin docs for patching Liferay DXP Digital Enterprise.

cc\ @zsoltbalogh @zoltantakacs

### CODE REVIEW

File: discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Patch types [](id=patch-types)
+
+Liferay ships Digital Enterprise fixes through three different channels. Fix packs and hotfixes. You can read about the differences in this document.
```
Comment (sez11a) [inline discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown]: You say three channels but then list two: fix pack and hotfixes. Is there another one, or should it be two channels?

File: discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## 2. Hotfix
+A hotfix is provided to a customer when they contact Liferay about an
+emergency situation, and Liferay's support team--working with the customer--
+determines that the problem is indeed an issue with the product that needs to be
+fixed very quickly. Support fixes the bug and provides a hotfix to the customer
+immediately. This is a short-term fix that solves the issue for the customer as
+quickly as possible. Hotfixes can patch both the core and the application
+suites.
+
+## Testing methods of patches
```
Comment (sez11a) [inline discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown]: Recommend cutting this section out. It seems germane to internal processes, but not to customers.

File: discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patching-tool-configuration/01-configuration-basic.markdown [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Patching Tool
```
Comment (sez11a) [inline discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patching-tool-configuration/01-configuration-basic.markdown]: When I go through this, I'll probably re-work some of these headings according to our [guidelines](https://github.com/liferay/liferay-docs/blob/master/guidelines/writers-guidelines.markdown).

File: discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown (hunk 2 above)
Comment (zsoltbalogh) [inline discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown]: We receive a lot of questions about testing this is why we added this information to the documentation. I've restructured it a bit and removed the details about unit and other types of testing.

File: discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown (hunk 1 above)
Comment (zsoltbalogh) [inline discover/deployment/articles-dxp/03-maintaining-liferay/01-patching-liferay/01-patch-types.markdown]: I forgot to save what I added before commit and the service packs were never added. I've pushed some changes to my branch. Could you please take it as the base for the review?

**Comment by zsoltbalogh** [comment]
@sez11a, I'm sending you another PR, I have not saved some files before sending and I also incorporated your feedback. Please close this one.

**Comment by jhinkey** [comment]
Closing per @zsoltbalogh's request.

**Pull request closed without merging**